docs/HISTORY.txt

```diff
@@ -1,13 +1,21 @@
 Changelog
 =========
 
-1.1.3 (unreleased)
+1.2.0 (unreleased)
```

Comment:
Do you know if this is going to work when using zest.releaser? I agree that it is a great idea to put it there if we already know that this should be a minor release (especially if we don't release it immediately).

Reply:
Let's see :-)